index.html

```diff
@@ -190,7 +206,8 @@ <h1>About me</h1>
                 <li>Gitlab</li>
                 <li>Terminal</li>
             
-        </li>
+            </ul>
+    </div>
     </div>
 </section>
 <div class="contact-form">
```

Comment:
- [ ] KIndly remove this extra space below your contact section so as to make your app user friendly 👍🏽 
![image](https://user-images.githubusercontent.com/63473584/215234281-74a3854e-4160-4b5c-8321-ed9d6d7f3b1f.png)